internal/embedding/data.go

```diff
@@ -7,14 +7,19 @@ import (
 	"runtime"
 )
 
-// CelestiaAppV3 returns the compressed platform specific Celestia binary.
-func CelestiaAppV3() ([]byte, error) {
+// NOTE: This version must be updated at the same time as the version in the
+// Makefile.
+const CELESTIA_V3_VERSION = "v3.10.0-arabica"
```

Comment:
mega nit, consts in golang usually follow camel casing

Reply:
Agreed. Oops, I copied this over from the Makefile but forgot to fix the casing.